src/geode/basic/identifier.cpp

```diff
@@ -34,7 +34,13 @@ namespace geode
     class Identifier::Impl
     {
     public:
-        Impl() = default;
+        Impl()
+        {
+            if( name_ == DEFAULT_NAME )
+            {
```

Comment:
any motivation for that change?
This name will persist even if the uuid changes... I would not do that.